### DISCUSSION TITLE
我保存到硬盘上的缓存文件，为什么会变小呢？

### DISCUSSION THREAD

**Discussion by yyysuo** [pymumu/smartdns#1744] [Q&A]
完整的配置文件如下，因为过期缓存的TTL超级长，理论上所有的记录都应该都缓存且不清理，硬盘上的缓存文件
应该越来越大才对，但是现在在未重启进程的情况下，缓存文件有变小的现象，是我哪里设置的不对吗？

server-name smartdnsfallback
dualstack-ip-selection no
serve-expired yes
cache-size 1000000
log-size 64K
log-num 1
log-level error
audit-size 64K
audit-num 1
cache-persist yes
serve-expired-ttl 259200000
cache-checkpoint-time 3600
cache-file /etc/smartdnsfalback/smartdns.cache
bind :6666
bind-tcp :6666 
proxy-server socks5://10.10.10.4:7891 -name socks5
server-tls 1.1.1.1 -proxy socks5
server-tls 1.0.0.1 -proxy socks5
server-tls 8.8.8.8 -proxy socks5
server-tls 8.8.4.4 -proxy socks5

**Comment by PikuZheng** [comment]
>serve-expired-ttl 259200000

不起作用，不允许这么大的数值

**Reply by yyysuo** [reply]
> > serve-expired-ttl 259200000
> 
> 不起作用，不允许这么大的数值

这么惨？最大是多少。

**Reply by PikuZheng** [reply]
c语言int类型的最大值，32767应该是

**Reply by yyysuo** [reply]
> c语言int类型的最大值，32767应该是

https://pymumu.github.io/smartdns/config/cache/#_5

看文档，应该能搞挺大的才对。
